sklearn/ensemble/tests/test_bagging.py

```diff
@@ -143,15 +144,19 @@ def fit(self, X, y):
     X_test_sparse = sparse_container(X_test)
     # Trained on sparse format
     sparse_classifier = BaggingClassifier(
-        estimator=CustomSVC(kernel="linear", decision_function_shape="ovr"),
+        estimator=CustomSVC(
+            probability=True, kernel="linear", decision_function_shape="ovr"
+        ),
         random_state=1,
         **params,
     ).fit(X_train_sparse, y_train)
     sparse_results = getattr(sparse_classifier, method)(X_test_sparse)
 
     # Trained on dense format
     dense_classifier = BaggingClassifier(
-        estimator=CustomSVC(kernel="linear", decision_function_shape="ovr"),
+        estimator=CustomSVC(
+            probability=True, kernel="linear", decision_function_shape="ovr"
```

Comment:
Same comment here. This test does should not use the deprecated attribute. If the tests need a base estimator with `predict_proba` we should instead wrap it with `CalibratedClassifierCV`.